### PR TITLE
Add syntax highlighting to README

### PR DESCRIPTION
**Pull request by pluma**
You can add syntax highlighting to code fence blocks by adding "js" (or "json" for JSON code examples) to the beginning of the code fence.

### CODE REVIEW

**Comment by philolo1** [comment]
@pluma thanks. Is jsx also possible?

**Comment by philolo1** [comment]
@pluma  please change it to `jsx` and i will merge it.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You need to install `react`, `react-dom`, `onsenui` and `react-onsenui`. You pro
 
 Now you can import the necessary libraries in your code:
 
-```
+```js
```
Comment (philolo1) [inline README.md]: please use `jsx` instead of `js` so all the xml tags will be colored, too!

**Comment by pluma** [comment]
Done.

**Comment by philolo1** [comment]
@pluma thanks!

**Comment by pluma** [comment]
You're welcome!

**Pull request merged**